src/gui/sdl_mapper.cpp

```diff
@@ -2035,6 +2038,49 @@ static void SetActiveBind(CBind *new_active_bind)
 	update_active_bind_ui();
 }
 
+// Drop other bound events for the given primary binding to ensure a given host
```

Comment:
What does "primary binding" mean? I've searched `sdl_mapper.cpp` for the term, and it looks like it's a new concept you introduced.

Is this new concept necessary, and if yes, would you explain it in a comment? Can't we just call it "binding" and that's it?